code/modules/holiday/halloween/jacqueen.dm

```diff
@@ -0,0 +1,442 @@
+					progression["[C.real_name]"] = progression["[C.real_name]"] | JACQ_DATE
+					visible_message("<b>[src]</b> looks to the side, deep in thought.</span>")
+					dating_start(C, gender)
+
+				if("Nevermind")
+					visible_message("<b>[src]</b> shrugs, <span class='spooky'>\"Suit yerself then, here's your candy back.\"</span>")
+					new /obj/item/reagent_containers/food/snacks/special_candy(loc)
+
+
+/mob/living/simple_animal/jacq/proc/trick(mob/living/carbon/C, gender)
+	var/option
+	if(ishuman(C))
+		option = rand(1,7)
+	else
+		option = rand(1,6)
+	switch(option)
+		if(1)
+			visible_message("<b>[src]</b> waves their arms around, <span class='spooky'>\"Hocus pocus, making friends is now your focus!\"</span>")
+			var/datum/objective/brainwashing/objective = pick("Make a tasty sandwich for", "Compose a poem for", "Aquire a nice outfit to give to", "Strike up a conversation about pumpkins with", "Write a letter and deliver it to", "Give a nice hat to")
+			var/mob/living/L2 = pick(GLOB.player_list)
+			objective += " [L2.name]."
+			brainwash(C, objective)
+		if(2)
+			visible_message("<b>[src]</b> waves their arms around, <span class='spooky'>\"Off comes your head, a pumpkin taking it's stead!\"</span>")
+			C.reagents.add_reagent("pumpkinmutationtoxin", 5)
+		if(3)
+			visible_message("<b>[src]</b> waves their arms around, <span class='spooky'>\"If only you had a better upbringing, your ears are now full of my singing!\"</span>")
+			var/client/C2 = C.client
+			C2.chatOutput.sendMusic("https://a.uguu.se/rQ8FxxUQ1Xzc_SpOwOkyOwOkyPumpkinSong-PFrPrIxluWk.mp4", 1)//I hope this works!
+		if(4)
+			visible_message("<b>[src]</b> waves their arms around, <span class='spooky'>\"You're cute little bumpkin, On your head is a pumpkin!\"</span>")
+			if(C.head)
+				var/obj/item/W = C.head
+				C.dropItemToGround(W, TRUE)
+			var/jaqc_latern = new /obj/item/clothing/head/hardhat/pumpkinhead/jaqc
+			C.equip_to_slot(jaqc_latern, SLOT_HEAD, 1, 1)
+		if(5)
+			visible_message("<b>[src]</b> waves their arms around, <span class='spooky'>\"In your body there's something amiss, you'll find it's a chem made by my sis!\"</span>")
+			C.reagents.add_reagent("eigenstate", 30)
+		if(6)
+			visible_message("<b>[src]</b> waves their arms around, <span class='spooky'>\"A new familiar for me, and you'll see it's thee!\"</span>")
+			C.reagents.add_reagent("secretcatchem", 30)
+		if(7)
+			visible_message("<b>[src]</b> waves their arms around, <span class='spooky'>\"While you may not be a ghost, for this sheet you'll always be it's host.\"</span>")
+			var/mob/living/carbon/human/H = C
+			if(H.wear_suit)
+				var/obj/item/W = H.wear_suit
+				H.dropItemToGround(W, TRUE)
+			var/ghost = new /obj/item/clothing/suit/ghost_sheet/sticky
+			H.equip_to_slot(ghost, SLOT_WEAR_SUIT, 1, 1)
+	poof()
+
+//Blame Fel
```

Comment:
**W O W**, calling me out like this, Fermi?

Reply:
Fel is big strong and can take the bullies for me